### PR TITLE
Renaming Cosmos DB Trigger input parameter to "documents"

### LINKED ISSUE #576: Rename trigger 'input' parameter to 'documents'
https://github.com/Azure/azure-webjobs-sdk-templates/blob/6094a10e330b8e492088b23f7148e7b2aa96397d/Functions.Templates/Templates/CosmosDBTrigger-CSharp/CosmosDBTriggerCSharp.cs#L24

We've seen confusion with the `input` parameter of the Cosmos DB trigger. Customers were confusing it with a Cosmos DB input binding. Can't we just rename this to `documents`?

### PR DESCRIPTION
**Pull request by ealsur**
This PR fixes #576 by renaming the variable that holds the documents that triggered the Function.

### CODE REVIEW

File: Functions.Templates/Templates/CosmosDBTrigger-CSharp/CosmosDBTriggerCSharp.cs [hunk 1]
```diff
@@ -21,12 +21,12 @@ public static class CosmosDBTriggerCSharp
             databaseName: "DatabaseValue",
             collectionName: "CollectionValue",
             ConnectionStringSetting = "ConnectionValue",
-            LeaseCollectionName = "leases")]IReadOnlyList<Document> input, TraceWriter log)
+            LeaseCollectionName = "leases")]IReadOnlyList<Document> documents, TraceWriter log)
```
Comment (soninaren) [inline Functions.Templates/Templates/CosmosDBTrigger-CSharp/CosmosDBTriggerCSharp.cs]: Can you update the variable name in line 7 
` public static void Run(IReadOnlyList<Document> input, TraceWriter log)`

Comment (ealsur) [reply Functions.Templates/Templates/CosmosDBTrigger-CSharp/CosmosDBTriggerCSharp.cs]: Done

**Comment by ealsur** [comment]
@soninaren I ran both building tools and checked that the renaming was done in the Portal and VS package (unzipped the nupkg to verify that for VS).

**Comment by soninaren** [comment]
@ealsur thanks

**Pull request merged**